server/test/handlers/cancelSpecificAppointment.test.js

```diff
@@ -0,0 +1,113 @@
+const workerTimeslot = require('../../models/db/workerTimeslot');
+const appointmentHandler = require('../../models/handlers/appointment');
+const workerTimeslotHandler = require('../../models/handlers/workerTimeslot');
+
+describe('testing cancellation of specific worker appointments', () => {
+    beforeEach(() => {
+        jest.resetModules();
+    });
+
+    test('no parameters passed in for the method', async () => {
+        // Arrange
+        const testIsCancelled = false;
+
+        // Act
+        const isCancelledSuccessfully = await appointmentHandler.cancelSpecificAppointment();
+
+        // Assert
+        expect(isCancelledSuccessfully).toBe(testIsCancelled);
+    });
+
+    test('null values for appointmentId parameter', async () => {
+        // Arrange
+        const appointmentId = null;
+        const testIsCancelled = false;
+
+        // Act
+        const isCancelledSuccessfully = await appointmentHandler.cancelSpecificAppointment(appointmentId);
+
+        // Assert
+        expect(isCancelledSuccessfully).toBe(testIsCancelled);
+    });
+
+    test('empty string for the appointmentId', async () => {
+        // Arrange
+        const appointmentId = '';
+
```

Comment:
Hi Peter, 
I would suggest removing the unneeded blank line between these two variable declarations to reduce the file length for the test code. The same applies to the following tests as well.

Reply:
Resolved!